site2/docs/administration-zk-bk.md

```diff
@@ -180,6 +180,13 @@ You can configure BookKeeper bookies using the [`conf/bookkeeper.conf`](referenc
 
 The minimum configuration changes required in `conf/bookkeeper.conf` are as follows:
 
+> **Note**  
+> The configuration of journalDirectory and ledgerDirectories needs to be considered clearly, and it is very troublesome to change later.
```

Comment:
I think it might be better to keep the description aligned across different files. Maybe considering use global replacement to change the description in all docs ? Thanks ~